### PR TITLE
PR regarding constant IsueTemplates for all repos.

### PR DESCRIPTION
**Pull request by Saurabhpatil-dev**
Hello @Panquesito7  @krishguptadev ,

I am creating this PR regarding the issue number #211 .

Please review, and let me know if any changes are required.

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-name: Bug
-description: Report a bug found in the bot's source code
+name: 🐛 Bug
+description: Report an issue to help improve the project.
 labels: ["🛠 goal: fix"]
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/bug.yml:3]:
```suggestion
labels: ["bug"]
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 body:
   - type: textarea
     id: description
     attributes:
       label: Description
-      description: A brief description of the question or issue. Also, include what you tried and what didn't work
+      description: A brief description of the question or issue, also include what you tried and what didn't work
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/bug.yml:9]:
```suggestion
      description: A brief description of the question or issue. Also, include what you tried and what didn't work
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: "[DOCS] <description>"
+labels: ["📄 aspect: text"]
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/docs.yml:4]:
```suggestion
labels: ["documentation"]
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for LinkFree? Please suggest!
+title: "[FEATURE] <description>"
+labels: ["⭐ goal: addition"]
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/feature_request.yml:4]:
```suggestion
labels: ["enhancement"]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**